<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Admin Dashboard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/icons/bank.png</normaloff>:/icons/icons/bank.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(244, 241, 222);
	color: rgb(61, 64, 91);
}
QPushButton
{
	font: 10pt &quot;IRANYekan&quot;;
	background-color: rgb(224, 122, 95);
	color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 255, 255);
    border-radius: 6px;
}
QPushButton:hover
{
	background-color: rgb(242, 204, 143);
}
QPushButton#pushButton_logout
{
	background-color: rgb(129, 178, 154);
}
QPushButton#pushButton_totalb
{
	background-color: rgb(95, 121, 123);
}
QPushButton#pushButton_totalbissued
{
	background-color: rgb(95, 121, 123);
}
QPushButton#pushButton_totaluser
{
	background-color: rgb(95, 121, 123);
}
QPushButton:hover#pushButton_logout
{
	background-color: rgb(242, 204, 143);
}
QStatusBar
{
	color: rgb(143, 93, 93);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_top">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 64, 91);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>45</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;IRANYekan&quot;;</string>
     </property>
     <property name="text">
      <string>ATM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>75</y>
       <width>320</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;IRANYekan Light&quot;;</string>
     </property>
     <property name="text">
      <string>User DashBoard</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>771</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Select</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>771</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>100</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>100</number>
         </property>
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <item>
          <widget class="QPushButton" name="buttonDeposit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Deposit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonWithdraw">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Withdraw</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonTransactionHistory">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Transaction History</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonLogout">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Logout</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>buttonDeposit</tabstop>
  <tabstop>buttonTransactionHistory</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
